pkg/helpers/collections.go

```diff
@@ -18,3 +18,11 @@ func GetMapValues[M ~map[K]V, K comparable, V any](m M) []V {
 	}
 	return res
 }
+
+func CopyMap[M ~map[K]V, K comparable, V any](m M) M {
```

Comment:
to prevent future mishapps, I'd like to see `shallow` in its name

Reply:
done